metricbeat/module/system/process/process.go

```diff
@@ -91,6 +95,20 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		}
 	}
 
+	// setup network tracking
+	if config.TrackNetworkData {
+		m.Logger().Infof("Starting system/process network monitoring")
+		var err error
+		m.networkMonitoring, err = network.NewNetworkTracker()
+		if err != nil {
+			return nil, fmt.Errorf("error creating network tracker: %w", err)
+		}
+		err = m.networkMonitoring.Track(context.Background())
```

Comment:
Is context.Background correct? Is the only way for this to be cancelled the process exiting?

Reply:
So, I added a `Close()` implementation to the metricset, but since the reporter interface doesn't give us any kind of parent context, not sure what use we would get from our own.